Simplenote/Controllers/AccountDeletionController.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+class AccountDeletionController {
+    var successHandler: (() -> Void)?
+    var failureHandler: ((Int) -> Void)?
+
+    func requestAccountDeletion(_ user: SPUser) {
+        SPAppDelegate.shared().accountDeletionRequestDate = Date()
```

Comment:
I think we should set it only if the request was actually successful.

Reply:
Plus perhaps the caller should do this. Feels a bit odd to do it here?